crates/uplc/test_data/conformance/evaluation/builtin/semantics/constrData/constrData.uplc.expected

```diff
@@ -1 +1 @@
-parse error
```

Comment:
What's this? Seems suspicious. The source of truth for these is the Plutus repo, we shouldn't be editing this by hand. Why is this changing from parse error to something else?

Reply:
The source of truth for this file stems from a time when the PlutusData syntax was not formalized yet. Therefore the testcase for constrData was broken. This is just a fix to the testcase to the IMO intended way it should work.

I agree it would make sense to change the upstream acceptance tests in the plutus repo (if they have not been updated yet) also as per #783